### ISSUE TITLE
Plugins: Removed plugins still show in plugins list

### ISSUE THREAD

**Issue by bsessions85** [wordpress-mobile/WordPress-Android#7188]
### Expected behavior
A removed plugin should not show on the plugins page

### Actual behavior
After removing a plugin and leaving and returning to the plugins page, the plugin still shows in the list of installed plugins

### Steps to reproduce the behavior
The Plugins page doesn't update properly.

1. Go to Plugins page
2. Select a plugin and remove it.
3. You will be taken to the plugins page and the plugin you removed is gone..
4. Go back to the dashboard 
5. Go to Plugins page

The plugin you removed is still there and refreshing doesn't make it go away. If I check on desktop I can see that the plugin was actually removed.

##### Tested on [device], Android [version]
Samsung Galaxy S7, Android 7.0

**Comment by nbradbury** [comment]
@bsessions85 I'm not able to reproduce this. I followed the steps you gave but each time when I deleted a plugin it remained deleted. Any chance this is only happening with a specific plugin?

@oguzkocer Any thoughts on what might cause this on the FluxC side?

**Comment by bsessions85** [comment]
It happened on two different plugins that I deleted. Even after removing both of them and then going away from and returning to the plugins page they were both showing.

**Comment by oguzkocer** [comment]
@nbradbury This should have been fixed in the previous release with the addition of pull to refresh unless it was re-introduced again later. The issue is with the API. It takes a few seconds for the fetch request to reflect any changes (including removal of a plugin) which means if we fetch the data immediately after a removal, it'll be re-added to the DB. A pull to refresh would fix the issue after a few seconds.

We have reported this several times, but we haven't gotten a fix yet. /cc @elibud
